README.md

```diff
@@ -1 +1,72 @@
+
+The router server is exposed on public IP address that you can access through SSH on the standard port, also mongodb port 27017 open. You can access it for MongoDB data write and read.
+
+The config server replica set stores sharding cluster metadata. MongoDB suggests to use a replica set for the metadata store in the production environment, in case one of the config server is down, there will still be other 2 config server nodes offer the service.
+
+1 shard is a 4 node replica set. You can shard the data on the replica set. You can also add more replica sets into the sharding cluster.
+
+The nodes are under the same subnet 10.0.0.0/24. Except the router server, the other nodes only have private IP address.
+
+<img src="https://raw.githubusercontent.com/cjsingh8512/azure-cosmosdb-mongodbshardedcluster/users/chsi/images/Mongo Sharded Cluster.png" />
+
+## Important Notice
+1. Each VM of the shard uses raid0 to improve performance. The number and the size of data disks(setup raid0) on each shard VM are determined by yourself. However, there is number and size of data disks limit per the VM size. Before you set number and size of data disks, please refer to the link https://azure.microsoft.com/en-us/documentation/articles/virtual-machines-linux-sizes/ for the correct choice
+2. Mongo router server is enabled with SSL. The certificate used for SSL has the same subject name as the FQDN of the server. Therefore, if you want to connect to the server from a driver or desktop client without enabling the setting **IngnoreInvalidHostnames** please use the FQDN as part of the connection string
+
+## After deployment, you can do below to verify if the sharding cluster really works or not:
+
+1. SSH connect to the router server, execute below:
+  ```
+  $mongo -u "<mongouser>" -p "<mongopassword>" "admin"
+
+  db.runCommand( { listshards : 1 } )
+
+  exit
+  ```
+
+  Upper db.runCommand( { listshards : 1 } ) command will show the sharding cluster details. 
+
+2. You can "shard" any database and collections you want. SSH connect to the router server, execute below:
+  ```
+  $mongo -u "<mongouser>" -p "<mongopassword>" "admin"
+
+  db.runCommand({enableSharding: "<database>" })
+
+  sh.status()
+
+  sh.shardCollection("<database>.<collection>", shard-key-pattern)
+
+  exit
+  ```
+
+3. You can add more shards into this sharding cluster. SSH connect to one of the router server, execute below:
+  ```
+  $mongo -u "<mongouser>" -p "<mongopassword>" "admin"
+
+  sh.addShard("<replica set name>/<primary ip>:27017")   
+
+  exit
+  ```
+
+  Before adding your own replica set into the sharding cluster, you should enable internal authentication in your replica set first, and make sure the replica set is accessible through this sharding cluster.
+
+## Known Limitations
+- The MongoDB version is 3.6.
```

Comment:
Why is this a limitation (server version 3.6)

Reply:
Actually none of the points mentioned are limitations. I have changed the title to Notes

Reply:
Makes sense. Please update